lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -467,6 +467,33 @@ namespace Js
         return typeId == TypeIds_Array;
     }
 
+    template<typename T>
+    bool JavascriptArray::IsMissingItemAt(uint32 index) const
+    {
+        SparseArraySegment<T>* headSeg = (SparseArraySegment<T>*)this->head;
+
+        return SparseArraySegment<T>::IsMissingItem(&headSeg->elements[index]);
+    }
+
+    bool JavascriptArray::IsMissingItemForUnkownArrayTypeAt(uint32 index)
```

Comment:
This is somewhat inconsistent naming with the rest of the array APIs.  Just call that one IsMissingItemAt() and leave the template one as is, or rename to something like DirectIsMissingItemAt().